test/test_helper.rb

```diff
@@ -13,5 +13,25 @@ class TestCase
     fixtures :all
 
     # Add more helper methods to be used by all tests here...
+    def setup_omniauth_mock
```

Comment:
使われ方を見ると

```suggestion
    def get_omniauth_mock
```
や
```suggestion
    def generate_omniauth_mock
```
なのかなと思いました。

Reply:
返却しているわけじゃないので、get ではなさそうですね。
じゃあ、generate かな〜

Reply:
はいー。

```ruby
Rails.application.env_config['omniauth.auth'] = setup_omniauth_mock
```

で違和感があったのでコメントしましたが、後でも良いと思います。

Reply:
違和感どおり、おかしいですね。
修正しました。